index.html

```diff
@@ -354,186 +338,198 @@
+
+<p dfn-for='PerformanceResourceTiming'>
+On getting, the <dfn>workerStart</dfn> attribute MUST return as follows:
+</p>
+<ol class='algorithm'>
+<li>
+<p>If the current browsing or worker context's have an <a href="https://www.w3.org/TR/service-workers/#dfn-containing-service-worker-registration">active worker</a> [[!SERVICE-WORKERS]]:</p>
+<ol>
+<li>
+the time immediately before the user agent <a href="https://www.w3.org/TR/service-workers/#on-fetch-request-algorithm">fires an event named `fetch`</a> at the <a href="https://www.w3.org/TR/service-workers/#dfn-active-worker">active worker</a> if the worker is available.
+</li>
+<li>
+the time immediately before the user agent <a href="https://www.w3.org/TR/service-workers/#service-worker-concept">runs the worker</a> required to service the request.</li>
+</ol>
+</li>
+<li>zero, otherwise.</li>
+</ol>
+
+<p dfn-for='PerformanceResourceTiming'>
+On getting, the <dfn>redirectStart</dfn> attribute MUST return as follows:
+</p>
+<ol class='algorithm'>
+<li>
+The time immediately before the user agent starts to <a href="https://www.w3.org/TR/html5/infrastructure.html#fetch">fetch</a> the resource that initiates the redirect, if there are HTTP redirects or <a href="https://www.w3.org/TR/html5/infrastructure.html#concept-http-equivalent-codes" title='HTTP response codes equivalence'>equivalent</a>
+when <a href="https://www.w3.org/TR/html5/infrastructure.html#fetch" title='fetch'>fetching</a> the resource and <strong>all</strong> the redirects or equivalent pass the <a>timing allow check</a> algorithm.
+</li>
+<li>zero, otherwise.</li>
+</ol>
+
+<p dfn-for='PerformanceResourceTiming'>
+On getting, the <dfn>redirectEnd</dfn> attribute MUST return as follows:
+</p>
+<ol class='algorithm'>
+<li>The time immediately after receiving the last byte of the response of the last redirect, if there are HTTP redirects or <a href="https://www.w3.org/TR/html5/infrastructure.html#concept-http-equivalent-codes" title='HTTP response codes equivalence'>equivalent</a>
+when <a href="https://www.w3.org/TR/html5/infrastructure.html#fetch" title='fetch'>fetching</a> the resource and <strong>all</strong> the redirects or equivalent pass the <a>timing allow check</a> algorithm.</li>
+<li>zero, otherwise.</li>
+</ol>
+
+<p dfn-for='PerformanceResourceTiming'>
+On getting, the <dfn>fetchStart</dfn> attribute MUST return as follows:
+</p>
+<ol class='algorithm'>
+<li>
+The time immediately before the user agent starts to <a href="https://www.w3.org/TR/html5/infrastructure.html#fetch">fetch</a> the <em>final</em> resource in the redirection, if there are HTTP redirects or <a href="https://www.w3.org/TR/html5/infrastructure.html#concept-http-equivalent-codes" title='HTTP response codes equivalence'>equivalent</a>.

```

Comment:
Aside: fwiw, I'm finding that pulling out common links into the footer and then using `[]`'s shorthand cleans up the spec quiet a bit and improves (editor) readability.